includes/template-functions.php

```diff
@@ -1097,3 +1097,28 @@ function edd_get_bundle_item_price_id( $bundle_item ) {
 
 	return $bundle_price_id;
 }
+
+/**
+ * Load a template file for a single download item.
+ *
+ * This is a wrapper function for backwards compatibility so the
+ * shortcode's attributes can be passed to the template file via
+ * a global variable.
+ *
+ * @since 2.9.0
+ *
+ * @param array $atts The [downloads] shortcode attributes.
+ * @param int   $i The current item count.
+ */
+function edd_download_shortcode_item( $atts, $i ) {
+	global $edd_download_shortcode_item_atts, $edd_download_shortcode_item_i;
```

Comment:
This last global should be `$edd_download_shortcode_item_$i`.

As it stands all items will be stored in the same global b/c it's using `i` as a string.